src/components/ProviderCard.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
 import Card from 'react-bootstrap/Card'
-import ListGroup from 'react-bootstrap/ListGroup'
+import { Route, Link } from 'react-router-dom'
+import Provider from '../pages/Provider'
 
 const ProviderCard = ({ data }) => {
-  const { credential, name, gender } = data.basic
+  const { credential, first_name, last_name, gender } = data.basic
 
   function titleCase(str) {
```

Comment:
May want to move this into a separate helpers file, as this is a generic function and not intrinsic to the functionality of ProviderCard